config/pyramid/ini/loader.ini

```diff
@@ -34,8 +34,8 @@ in_docker = true
 cors_trusted_origins =
     http://localhost:3000
 file_upload_bucket = igvf-files-local
+restricted_file_upload_bucket = igvf-restricted_files-local
```

Comment:
Need to fix `igvf-restricted_files-local` -> `igvf-restricted-files-local`

Reply:
done